src/shogun/machine/LinearMulticlassMachine.h

```diff
@@ -161,23 +132,17 @@ class LinearMulticlassMachine : public MulticlassMachine
 		 */
 		virtual void add_machine_subset(SGVector<index_t> subset)
 		{
-			/* changing the subset structure to use subset stacks. This might
-			 * have to be revised. Heiko Strathmann */
-			m_features->add_subset(subset);
+			
 		}
 
 		/** deletes any subset set to the features of the machine */
 		virtual void remove_machine_subset()
 		{
-			/* changing the subset structure to use subset stacks. This might
-			 * have to be revised. Heiko Strathmann */
-			m_features->remove_subset();
+		
```

Comment:
if these methods becomes noops can they just be removed? How are the subsets handled now?

Reply:
just add/remove subset to feature that will be passed in `train_machine`.

Reply:
I see, makes sense to me :)